src/scales/scale.category.js

```diff
@@ -53,10 +53,25 @@ module.exports = function() {
 
 		getLabelForIndex: function(index, datasetIndex) {
 			var me = this;
-			var data = me.chart.data;
+			var chart = me.chart;
+			var data = chart.data;
 			var isHorizontal = me.isHorizontal();
 
-			if (data.yLabels && !isHorizontal) {
+			// Chart might be undefined or not contain getDatasetMeta function.
+			// This is the case in some tests, and probably only in tests
+			var ds = chart.getDatasetMeta && chart.getDatasetMeta(datasetIndex).controller;
```

Comment:
Instead, we should fix these tests by creating a chart instance and avoid this check in our sources since `getDatasetMeta` should never be undefined.

Reply:
@kurkle just a gentle reminder about this comment